pywinauto/UIAElementInfo.py

```diff
@@ -167,29 +167,46 @@ def parent(self):
         else:
             return None
 
-    @property
-    def children(self):
-        "Return list of immediate children for the element"
-        children = []
-        
-        childrenArray = self._element.FindAll(_tree_scope['children'], _true_condition)
-        for childNumber in range(childrenArray.Length):
-            childElement = childrenArray.GetElement(childNumber)
-            children.append(UIAElementInfo(childElement))
+    def _get_elements(self, scope, cond=_true_condition):
+        "Find all elements according to the given scope and conditions"
+        elements = []
+        ptrs_array = self._element.FindAll(scope, cond)
+        for num in range(ptrs_array.Length):
+            child = ptrs_array.GetElement(num)
+            elements.append(UIAElementInfo(child))
+
+        return elements
+
+    def _build_condition(self, process = None, class_name = None, title = None):
```

Comment:
Not exactly. The single underscore is for non-public interfaces of any kind: attributes, stand-alone functions. It just means the interface is for the internal usage only 

> 

> Non-public attributes are those that are not intended to be used by third parties; you make no guarantees that non-public attributes won't change or even be removed.

 The double forward underscore, as you said, is a sort of "sealing" of the method and preventing its sub-classing. Not sure that's the goal here. Basically, UIAElementInfo isn't supposed to be sub-classed but we do not prohibit it.